dev/com.ibm.ws.app.manager.springboot/src/com/ibm/ws/app/manager/springboot/util/SpringBootThinUtil.java

```diff
@@ -430,7 +432,8 @@ public static StarterFilter getStarterFilter(Stream<String> entries) {
             if (version.get() == null || container.get() == null) {
                 String path = entry;
                 if (version.get() == null && path.contains("BOOT-INF/lib/spring-boot-") && path.endsWith(".jar")) {
```

Comment:
How does this work for WARs where the `spring-boot-` JARs would be in `WEB-INF/lib/` folder?

Reply:
I think this code was meant to filter starter jars from a JAR application.My understanding is that a WAR ideally wouldn't have starter jars and it will be specified as provided dependency in pom.xml. Please let me know if I should modify the code to add WEB-INF/lib folder as well.